diag_manager/fms_diag_yaml.F90

```diff
@@ -484,8 +484,8 @@ function get_total_num_vars(diag_yaml_id, diag_file_ids) &
 subroutine check_file_freq(fileobj)
   type(diagYamlFiles_type), intent(inout) :: fileobj      !< diagYamlFiles_type obj to check
 
-  if (fileobj%file_freq < 1 ) &
-    call mpp_error(FATAL, "freq must be greater than 0. &
+  if (fileobj%file_freq < -1 ) &
+    call mpp_error(FATAL, "freq must be greater than -1. &
```

Comment:
This checks if `fileobj%file_freq` is less than -1.  If it IS -1, then this will be false.  Your error message says that it must be greater than -1.
Either this should be
```fortran
if (.not.(fileobj%file_freq > -1)) 
```
or your error message needs to be changed.

Reply:
I changed it to: 
https://github.com/NOAA-GFDL/FMS/blob/011145d0ae844d01b7f1a474d8420ba3cd803137/diag_manager/fms_diag_yaml.F90#L487-L488

If the file_freq is not greater than on equal to -1, it should fail because freq must be greater than or equal to -1. 
It is easier to read. 

-1 is a valid freq: 
https://github.com/NOAA-GFDL/FMS/blob/2c218d0aabd5340bde8f157d15ade6efd128e163/diag_manager/diag_table.F90#L79-L86